app/helpers/session_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module SessionHelpers
+  def current_user
+    User.find(session[:id])
+  end
+
+  def is_logged_in?
+    !!session[:id]
```

Comment:
It's better to rely on the successful call to `current_user` than only introspecting on the `session[:id]` value. In theory, a user might have an invalid `id` in their session which would make `logged_in?` return true even though that user would not be findable later. Also, because `current_user` is memoized, it makes sense to call it as early in the request cycle as possible, which generally is the first call `logged_in?`.

```ruby
def logged_in?
  !!current_user
end
```